src/utp_wrapper.rs

```diff
@@ -47,7 +49,13 @@ impl UtpWrapper {
                                     }
                                 },
                                 Err(TryRecvError::Disconnected) => break 'outer,
-                                Err(TryRecvError::Empty) => break,
+                                Err(TryRecvError::Empty) => {
+                                    if timeout < ::time::SteadyTime::now() {
```

Comment:
> But I think that is design issue to decide whether it is valid or not.

I hardly think so. I can write a program to download a 10GB file from an HTTP server that will only, and only, read bytes after the initial HTTP boilerplate is written. This behaviour isn't wrong and shouldn't be forbidden.

> The purpose of this PR is to resolve the hanging issue of the test, so the test itself can be included in the CI and to be used to verify the utp related work.

Isn't there another way to achieve it?